supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -21,3 +21,72 @@ When the format _day Month year_ causes a presentation or clarity issue, use ins
 
 [[product-names-versions-ref]]
 == Product names and version references
+
+////
+PROPOSED ADDITIONS?
+master-docinfo.xml specific and portal-specific version variables
+URL fragments and composed links?
+correct sequence when combining attributes
+ensure proper sequence for compound attributes
+syntax recommendations: what separators to use? dashes/hyphens, underscores?
+////
+
+Whenever you refer to the name of your product in full, or in its abbreviated form, or when you refer to the major and minor version of your product, avoid using hard-coded references and use attributes instead.
+
+Only use hard-coded version references if the version that you are referring to in that particular case never changes.  
+
+=== Attribute file
+
+Define attributes for product name and product version in a single common shared attributes file for each documentation set or product.
+
+Name the file `attributes.adoc`.
+You may use a different name if necessary, but if you do, ensure that the name of the file indicates to your collaborators that it is an attribute file.
+
+//For a single title, store the file in the same directory as the `master.adoc` file.
+For sharing between multiple titles, store the file in a common directory above the directories that contain your titles, such as `docs/common/` or `docs/topics/templates`.
+
+Include the attributes file at the beginning of the `master.adoc` files of all titles in your documentation set:
+
+.master.adoc
+[source,asciidoc,options="nowrap"]
+----
+\include::common/attributes.adoc[]
+----
+
+TIP: Create a symbolic link pointing to the `common` directory inside the directory containing the `master.adoc` file of your title. It makes including the attribute file in your title easier. +
+Do not copy the directory or attribute file directly, and do not use hard links!
+
+=== Minimum required attributes
+
+For each title ensure that you define attributes for the following values:
+
+* product name
```

Comment:
I think it would be clearer to make this a description list and use the actual syntax. If you're worried about teams using differing syntax, you can add a note to that effect before or after the list. You could also make the topic more concise that way. In fact, I think you could combine the _Minimum required attributes_ and _Attribute usage_ in one section.

Reply:
Thanks, @stoobie that could work.
Originally, I stayed away from specific examples precisely because I wanted to avoid inflaming a dispute about whether I am recommending a specific syntax for these attributes or not. 
In light of the comments that I have seen from other reviewers, it might be best to not recommend a specific syntax for attributes in this guide at all.
Would it make sense to have the recommended usage of attributes (incl. file name, location, etc, ...) covered in another guide (like the mod docs guidelines?) That way we can cover some of the interesting cases related to reuse such as `ProductName` resolving to different product names while used in the same title.

Reply:
@inoxx03 +1